tests/timer/test_Event.cpp

```diff
@@ -0,0 +1,68 @@
+#include "gtest/gtest.h"
+
+#include "Kokkos_Core.hpp"
+
+#include "kokkos-utils/timer/Event.hpp"
+
+/**
+ * @file
+ *
+ * @addtogroup unittests
+ *
+ * Events in time
```

Comment:
I don't like "Event in time". Not sure what could be used instead though.

Reply:
One possibility could be to rename "Event" everywhere as "TimePoint". We'd avoid confusion with the "callback events". And we'd follow `std::time_point`. Let's discuss about it before I change everything.

Reply:
OK. Made it just "event" everywhere.